W-19141129 aws secrets manager json dots (#2990)
mulesoft/docs-connectors: amazon-secrets-manager-properties-provider/1.0/modules/ROOT/pages/amazon-secrets-manager-examples.adoc

	</s3:config>
----

=== Handle JSON Keys with Dots

> When your JSON secret contains keys with dots in their names, you need to handle them differently because DataWeave treats dots as subtrees in JSON. Ensure that configuration fields are set to `Expression mode` (fx button enabled) when using these syntaxes.

Reviewer: ```suggestion
If your JSON secret contains keys with dots in their names, DataWeave treats the dots as subtrees in JSON. Make sure that configuration fields are set to `Expression mode` (fx button enabled) when using these syntaxes.
```
